lib/train/transports/docker.rb

```diff
@@ -69,18 +69,27 @@ def close
       # nothing to do at the moment
     end
 
-    def file(path)
-      @files[path] ||=\
-        if os.aix?
-          Train::File::Remote::Aix.new(self, path)
-        elsif os.solaris?
-          Train::File::Remote::Unix.new(self, path)
-        else
-          Train::File::Remote::Linux.new(self, path)
-        end
+    def uri
```

Comment:
Is there a reason to make this public?

Reply:
I think we need it for now as inspec calls it here: https://github.com/chef/inspec/blob/master/lib/inspec/rspec_json_formatter.rb#L561

Reply:
Missed it, thank you.